aten/src/ATen/native/cpu/IndexKernel.cpp

```diff
@@ -116,9 +118,9 @@ struct IndexToOffset {
   const IntArrayRef sizes;
   const IntArrayRef strides;
   const int ndim;
-  IndexToOffset(const Tensor & tensor) : sizes(tensor.sizes()),
-                                         strides(tensor.strides()),
-                                         ndim(tensor.dim()) {}
+  IndexToOffset(const TensorBase & tensor) : sizes(tensor.sizes()),
```

Comment:
internal lint is complaining about this needing to be marked explicit. I know it's not your code but might as well make it happy.